test/test_interface.jl

```diff
@@ -0,0 +1,33 @@
+using LieGroups, Test
+
+s = joinpath(@__DIR__, "LieGroupsTestSuite.jl")
+!(s in LOAD_PATH) && (push!(LOAD_PATH, s))
+using LieGroupsTestSuite
+
+@testset "Generic Lie Group Interface functions" begin
+    M = LieGroupsTestSuite.DummyManifold()
+    op = LieGroupsTestSuite.DummyOperation()
+    G = LieGroup(M, op)
+    rs = "LieGroup(LieGroupsTestSuite.DummyManifold(), LieGroupsTestSuite.DummyOperation())"
+    @test repr(G) == rs
+    𝔤 = LieAlgebra(G)
+    op2 = LieGroupsTestSuite.DummySecondOperation()
+    rs2 = "Lie Algebra( LieGroup(LieGroupsTestSuite.DummyManifold(), LieGroupsTestSuite.DummyOperation()) )"
+    @test repr(𝔤) == rs2
```

Comment:
```suggestion
    rs2 = "LieAlgebra(LieGroup(LieGroupsTestSuite.DummyManifold(), LieGroupsTestSuite.DummyOperation()))"
    @test repr(𝔤) == rs2
```
I think that's how it should be printed.

Reply:
I Fixed this in a separate commit, since it also required to do the fix in the corresponding show-method.